server/modules/mobile/service/arenaMobileImport/arenaMobileImportJob.js

```diff
@@ -0,0 +1,62 @@
+import { SRSs } from '@openforis/arena-core'
+
+import Job from '@server/job/job'
+
+import * as FileUtils from '@server/utils/file/fileUtils'
+
+import ArenaSurveyReaderJob from './jobs/arenaSurveyReaderJob'
+import RecordsImportJob from './jobs/recordsImportJob'
+import CreateRdbJob from './jobs/createRdb'
+
+const createInnerJobs = () => {
+  return [
+    new ArenaSurveyReaderJob(),
+    new RecordsImportJob(),
+    // Needed when the survey is published
+    new CreateRdbJob(),
+  ]
+}
+
+export default class ArenaMobileImportJob extends Job {
```

Comment:
instead of an ArenaMobileImportJob it would be better to have an ArenaRecordsImportJob that reuses the inner jobs already defined for the arenaImportJob (they could be updated, of course, and the survey export job should be updated accordingly)